package.json

```diff
@@ -63,5 +63,9 @@
   },
   "engines": {
     "node": "14.x"
+  },
+  "volta": {
+    "node": "14.19.1",
+    "yarn": "1.19.2"
```

Comment:
Can we have this at the following values please?

```
    "node": "16.16.0",
    "yarn": "1.22.17"
```

Reply:
Hi, Ankush 
I have a question, the branch volta-addition which is resolving PR #358 has package.json file with different dependency versions and our develop branch is having package.json with different ones. this is causing yarn.lock file to change after installing necessary modules. 
for example ->  "next": "^10.0.4" (In volta-addition branch) and "next": "^12.1.0"(In develop branch),
Solution -> after executing yarn, the changes in my yarn.lock file can fix this, can I also commit those changes?